bridgestyle/sld/fromgeostyler.py

```diff
@@ -303,8 +303,8 @@ def _lineSymbolizer(sl, graphicStrokeLayer=0):
             _addCssParameter(
                 stroke, "stroke-dasharray", "%s %s" % (str(fsize), str(finterval))
             )
-        except:
-            _addCssParameter(stroke, "stroke-dasharray", "10 10")
+        except TypeError:
+            pass
```

Comment:
Is it useful to keep this try/catch ?

Reply:
Yes it's useful for the `float()` function. I agree this is most probably not the best implementation, but since it's in the main code (Geostyler -> SLD) and we don't have any unit tests, I find it a bit risky to change this and I prefer to leave it this way